bin/rucio

```diff
@@ -1844,6 +1844,47 @@ def rse_completer(prefix, parsed_args, **kwargs):
     return ["%(rse)s" % rse for rse in client.list_rses()]
 
 
+def __s3_verify_bucket_path(path):
+    # TODO: use as a part of class RSEDeterministicTranslation(object)?
+    # bucket_path_pattern = r"^user\.[a-zA-Z0-9_]+:/[a-zA-Z0-9_/]+$"
+    bucket_path_pattern = r"^user\.[a-zA-Z0-9_]+:/[a-zA-Z0-9_/]+(/[\w.-]+)?$"
```

Comment:
@bari12 we should maybe have a more proper definition for this in the common constants, or the schema? on the one hand, why not let people do what they want with their object stores? on the other, it's kind of managed storage. i'm conflicted.